dlt/destinations/impl/snowflake/utils.py

```diff
@@ -94,7 +94,9 @@ def gen_copy_sql(
         if stage_bucket_url:
             parsed_bucket_url = urlparse(stage_bucket_url)
             stage_bucket_path = parsed_bucket_url.path.lstrip("/")
-            if stage_bucket_path:
+            if parsed_bucket_url.netloc and parsed_bucket_url.scheme:
+                relative_url = relative_url.removeprefix(f"{parsed_bucket_url.scheme}://{parsed_bucket_url.netloc}/")
```

Comment:
Duplicating [my comment](https://github.com/dlt-hub/dlt/issues/3011#issuecomment-3263960582):

This code most likely has no effect as `relative_url` is already `path` from parsed. See `relative_url = parsed_file_url.path.lstrip("/")` and `parsed_file_url = urlparse(file_url)` from above